core/modules/node/node.types.inc

```diff
@@ -163,11 +163,18 @@ function node_type_form($form, &$form_state, $type = NULL) {
     '#type' => 'radios',
     '#title' => t('Default status'),
     '#options' => array(
-      1 => t('Published'),
-      0 => t('Unpublished'),
+      NODE_PUBLISHED => t('Publish now'),
+      NODE_NOT_PUBLISHED => t('Save draft'),
+      NODE_SCHEDULED => t('Publish later'),
+    ),
+    '#default_value' => isset($type->settings['status_default']) ? $type->settings['status_default'] : NODE_PUBLISHED,
+    '#description' => t('Content saved as draft is only accessible to the content creator or site administrators.') . ' ' . t('Content set to Schedule for later will be saved as draft when the content is created, but will be automatically published on the date and time selected.'),
```

Comment:
"Schedule for later" isnt an option in the radio list. Need to change either the radio list or the description text.